beta-src/src/components/ui/WDMain.tsx

```diff
@@ -253,6 +258,78 @@ const WDMain: React.FC = function (): React.ReactElement {
       isLivePhase: false,
     };
   };
+
+  useEffect(() => {
+    Object.entries(ordersMeta).forEach(([_, orderMeta]) => {
+      // Let's use as an example ION C APU - TUN
+      // The board should automatically also enters APU - TUN (Via Convoy)
+      // More info in CHR-68
+      const orderMetaUpdate = orderMeta.update;
+
+      if (
+        orderMetaUpdate?.type === "Convoy" &&
+        orderMetaUpdate?.fromTerrID &&
+        orderMetaUpdate.toTerrID
+      ) {
+        const fromTerr = maps.terrIDToProvince[orderMetaUpdate.fromTerrID];
+        const toTerr = maps.terrIDToProvince[orderMetaUpdate.toTerrID];
```

Comment:
```
const fromTerr = maps.terrIDToProvince[orderMetaUpdate.fromTerrID];
const toTerr = maps.terrIDToProvince[orderMetaUpdate.toTerrID];
```
Typefully, these are Provinces, not Territories? It looks a bit unhygenic for code to be assigning a province to a variable named "fromTerr" or "toTerr" even if the type distinction doesn't happen to matter in a case.

Reply:
Good point, updated